spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -47,6 +47,22 @@ class PowerShellManager; end
       expect(result[:exitcode]).to eq(1)
     end
 
+    it "should return the exitcode of the last command to set an exit code" do
+      result = manager.execute("$LASTEXITCODE = 0; write-output 'foo'; cmd.exe /c 'exit 99'; write-output 'bar'")
+
+      # STDERR is interpolating the newlines thus it's \n instead of the usual Windows \r\n
+      expect(result[:stdout]).to eq("foo\r\nbar\r\n")
+      expect(result[:exitcode]).to eq(99)
```

Comment:
This might be an unexpected result now, but arguably it is "correct".

Worth discussing @puppetlabs/windows